src/services/users/signin-user-by-email.test.js

```diff
@@ -0,0 +1,35 @@
+const { expect } = require('chai')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const signInUserByEmail = require('./signin-user-by-email')
+
+describe('User signin', () => {
+	let user
+	let validPassword
+	before(async () => {
+		user = await fixtures.insertUser({
+			password : 'validPaswword123$'
+		})
+		validPassword = 'validPaswword123$'
```

Comment:
kindly bring the variable initialization before the insert User and use that so we won't need to hardcode twice

```suggestion
		validPassword = 'validPaswword123$'
		user = await fixtures.insertUser({
			password : validPassword
		})

```